modules/gdscript/tests/scripts/runtime/features/setter_getter_with_property_argument.gd

```diff
@@ -0,0 +1,29 @@
+var my_float : float
+var my_int : int
+
+var my_float_alias : float : set=set_property, get=get_property
+var my_int_alias : int : set=set_property, get=get_property
+
+var aliases := {"my_float_alias" : "my_float", "my_int_alias" : "my_int"}
+
+
+func set_property(property : StringName, value : Variant) -> void:
+	var dealiased_property : StringName = property if !aliases.has(property) else aliases[property]
+	set(dealiased_property, value)
+
+
+func get_property(property : StringName) -> Variant:
+	var dealiased_property : StringName = property if !aliases.has(property) else aliases[property]
+	return get(dealiased_property)
+
+
+func test():
+	my_float = 0.0
+	my_int = 1
+	print((my_float_alias == my_float) && (my_float == 0.0))
+	print((my_int_alias == my_int) && (my_int == 1))
+
+	my_float_alias = 2.0
+	my_int_alias = 3
+	print((my_float_alias == my_float) && (my_float == 2.0))
+	print((my_int_alias == my_int) && (my_int == 3))
```

Comment:
This file does not follow the GDScript style guide (wrong spaces, symbolic logical operators, missing trailing newline). This is undesirable in non-parser tests. In addition, I do not see the point in this test, in my opinion it duplicates the ones I have already added.

Reply:
Ah, I was going to ask you about this test on Rocket. I had written this a week or two ago before your patch. I noticed it was redundant with your tests. I can remove it.